frontend/src/employee-mobile-frontend/components/attendances/child-info/ChildSensitiveInfo.tsx

```diff
@@ -0,0 +1,238 @@
+
+          <ContentAreaWithShadow opaque={true}>
+            <CollapsibleSection
+              title={i18n.attendances.childInfo.allergiesHeader}
+            >
+              <FixedSpaceColumn>
+                {render(
+                  i18n.attendances.childInfo.allergies,
+                  child.allergies,
+                  'child-info-allergies'
+                )}
+
+                {render(
+                  i18n.attendances.childInfo.diet,
+                  child.diet,
+                  'child-info-diet'
+                )}
+
+                {render(
+                  i18n.attendances.childInfo.medication,
+                  child.medication,
+                  'child-info-medication'
+                )}
+              </FixedSpaceColumn>
+            </CollapsibleSection>
+          </ContentAreaWithShadow>
+
+          <ContentAreaWithShadow opaque={true}>
+            <CollapsibleSection
+              title={i18n.attendances.childInfo.contactInfoHeader}
+            >
+              {child.contact1 && (
+                <>
+                  <Title size={4}>{i18n.attendances.childInfo.contact1}</Title>
+                  <FixedSpaceColumn>
+                    <KeyValue>
+                      <Label>{i18n.attendances.childInfo.name}</Label>
+                      <P fitted data-qa={'child-info-contact1-name'}>{`${
+                        child.contact1.firstName || ''
+                      } ${child.contact1.lastName || ''}`}</P>
+                    </KeyValue>
+
+                    {render(
+                      i18n.attendances.childInfo.phone,
+                      child.contact1.phone,
+                      'child-info-contact1-phone'
+                    )}
+
+                    {render(
+                      i18n.attendances.childInfo.backupPhone,
+                      child.contact1.backupPhone,
+                      'child-info-contact1-backup-phone'
+                    )}
+
+                    {render(
+                      i18n.attendances.childInfo.email,
+                      child.contact1.email,
+                      'child-info-contact1-email'
+                    )}
+
+                    <Divider />
+                  </FixedSpaceColumn>
+                </>
+              )}
+
+              {child.contact2 && (
+                <>
+                  <Title size={4}>{i18n.attendances.childInfo.contact2}</Title>
+                  <FixedSpaceColumn>
+                    <KeyValue>
+                      <Label>{i18n.attendances.childInfo.name}</Label>
+                      <P fitted data-qa={'child-info-contact2-name'}>{`${
+                        child.contact2.firstName || ''
+                      } ${child.contact2.lastName || ''}`}</P>
+                    </KeyValue>
+
+                    {render(
+                      i18n.attendances.childInfo.phone,
+                      child.contact2.phone,
+                      'child-info-contact2-phone'
+                    )}
```

Comment:
The spec has icon telephone links for phone numbers.

Reply:
yes - sorry forgot this from the done - in - next - pr list, added